server/app/services/settings/SettingsManifest.java

```diff
@@ -940,6 +940,11 @@ public boolean getProgramMigrationEnabled(RequestHeader request) {
     return getBool("PROGRAM_MIGRATION_ENABLED", request);
   }
 
+  /** (NOT FOR PRODUCTION USE) Enables filtering programs by category on the homepage */
+  public boolean getProgramFiltering(RequestHeader request) {
+    return getBool("PROGRAM_FILTERING", request);
```

Comment:
nitpick (maybe blocking): I think the pattern we've established with feature flags is to include "enabled" in the name (see `PROGRAM_MIGRATION_ENABLED` above). I'm not sure exactly when we decided on this or why, but my guess is because it makes the function name super clear (eg. `getProgramFilteringEnabled`).

Reply:
I saw that about half of the flags end with "enabled", but I don't like that pattern because it makes it seem like the flag itself is always enabled.  Do we then ask ourselves "Is Program Migration Enabled enabled?"  It's redundant.  Maybe we should bring this up at the eng sync on Monday.  If we want to make the function name more clear we could include the word "Flag" at the end of each FF name.  Then, the function would be `getProgramFilteringFlag`.  And we could say "Is the Program Filtering Flag enabled?"

Reply:
I agree! Good idea to discuss on Monday.